test/local/schema.test.ts

```diff
@@ -0,0 +1,188 @@
+import { intentOrArraySchema, intentSegmentSchema, intervalConditionSchema, limitSwapActionSchema, marketSwapActionSchema, nonceConditionSchema, priceConditionSchema, replaySchema, TokenArgsSchema } from "@brink-sdk/intents/DSL/schema";
+import { expect } from 'chai';
+
+describe('Brink DSL Schema Tests', () => {
+  describe('TokenArgsSchema', () => {
+    it('validates a correct Ethereum address', () => {
+      const input = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
+      const result = TokenArgsSchema.validate(input);
+      expect(result.error).to.be.undefined;
+    });
+
+    it('throws error for an incorrect Ethereum address', () => {
+      const input = "0xInvalidEthereumAddress";
+      const result = TokenArgsSchema.validate(input);
+      expect(result.error).to.not.be.undefined;
```

Comment:
done. actually most tests check that it validates correct payloads.  There are so many cases where it can fail, i don't know if we should add them. but i'll add tests for the extensions since they're risky.